core/primitives/src/transaction.rs

```diff
@@ -104,6 +104,7 @@ pub struct TransferAction {
 #[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Clone, Debug)]
 pub struct StakeAction {
     pub stake: Balance,
+    /// A public key of a validator
```

Comment:
This comment seems unnecessary/confusing. Maybe say that it is a staking key/validator key